### ISSUE TITLE
Can't set rank higher than 32

### ISSUE THREAD

**Issue by BelieveDiffusion** [GraftingRayman/ComfyUI-ZImage-Bake-Extract#2]
Hello! I'm trying out the custom node, and I'm finding that I can't set the rank value higher than 32. Is that a hard limitation? I've found in the past that for complex LoRAs I've needed 64 or even 128 for success.

**Comment by BelieveDiffusion** [comment]
Similarly it looks you can't set alpha higher than 2.0. I've seen and used LoRAs with rank and alpha the same in the past.

**Comment by GraftingRayman** [comment]
Upped rank to 128 and alpha to 10.0 as the max

If you need it higher, let me know

**Comment by BelieveDiffusion** [comment]
Thank you! I'd suggest making them both 128, given that (as I understand it) it's valid to have a LoRA with the same alpha as rank.

**Comment by GraftingRayman** [comment]
I think i may have the alpha incorrect here, the alpha here in this node relates to the weight of the lora to be applied

**Comment by BelieveDiffusion** [comment]
Oh! Okay. So that gives a way to bake a LoRA into a base model at (say) 2x strength? If so, then yes, that's not what I was thinking of as alpha.

I was thinking more of the alpha used for the values in the output LoRA when extracting a LoRA from a base model and a fine-tuned model, where you specify both the rank and the alpha, and it's common for them to be the same:

https://www.reddit.com/r/LocalLLaMA/comments/1lrgdzg/is_there_a_rule_between_alpha_α_and_rank_r_for/

**Comment by GraftingRayman** [comment]
Correct, if alpha is 2.0, then it uses 2x the strength in baking/extracting

**Comment by BelieveDiffusion** [comment]
I think that's different to the alpha used for the values stored in the LoRA though, right? That's more of a multiplier / scaler for how the values are stored in the LoRA itself. (Showing my ignorance here, but I thought LoRA rank and alpha were different to LoRA application weight.)

**Comment by GraftingRayman** [comment]
I believe it is the same (though may get corrected by someone)

I have been training using musubi-trainer

Where network_dim = rank
And there is no network_alpha setting used

From my understanding alpha (Scaling Coefficient): Acts as a scaling factor that determines how strongly the LoRA affects the base model. Which in this case the alpha setting is doing this.

**Comment by BelieveDiffusion** [comment]
I've also been training with musubi-trainer, and it has both dim (rank) and alpha for LoRA training. They are `network_dim` and `network_alpha` respectively. The example config file here shows both: https://github.com/kohya-ss/musubi-tuner/blob/main/docs/advanced_config.md#using-configuration-files-to-specify-training-options--設定ファイルを使用した学習オプションの指定

**Comment by GraftingRayman** [comment]
Understood

However, the ZImage specific documentation, which I have been using to train has no such setting

https://github.com/kohya-ss/musubi-tuner/blob/main/docs/zimage.md

**Comment by BelieveDiffusion** [comment]
Oh! Does Z-Image not support it in musubi-tuner, or is it just not mentioned in the example and therefore uses a default?

**Comment by GraftingRayman** [comment]
Not sure, the only line in the logs I could find is below:

INFO:musubi_tuner.networks.lora:create LoRA network. base dim (rank): 64, alpha: 1

It does however have network_alpha in lora_zimage.py

```
def create_arch_network(
    multiplier: float,
    network_dim: Optional[int],
    network_alpha: Optional[float],
    vae: nn.Module,
    text_encoders: List[nn.Module],
    unet: nn.Module,
    neuron_dropout: Optional[float] = None,
    **kwargs,
):
```

and

```
    return lora.create_network(
        ZIMAGE_TARGET_REPLACE_MODULES,
        "lora_unet",
        multiplier,
        network_dim,
        network_alpha,
        vae,
        text_encoders,
        unet,
        neuron_dropout=neuron_dropout,
        **kwargs,
    )
```

With network_alpha being a float, I would think its just a multiplier like the weight of a lora
